src/api/handlers/guild.ts

```diff
@@ -120,16 +177,14 @@ export async function createGuildChannel(
 
   const result =
     (await RequestManager.post(endpoints.GUILD_CHANNELS(guild.id), {
-      ...options,
-      name,
-      permission_overwrites: options?.permissionOverwrites?.map((perm) => ({
-        ...perm,
-
-        allow: calculateBits(perm.allow),
-        deny: calculateBits(perm.deny),
-      })),
-      type: options?.type || ChannelTypes.GUILD_TEXT,
-    })) as ChannelCreatePayload;
+      ...keysToSnake(options),
+      type: options?.type
```

Comment:
`options?.type || ChannelTypes.GUILD_TEXT`

Reply:
This does not work